tools/pythonpkg/tests/conftest.py

```diff
@@ -5,24 +5,37 @@
 from os.path import abspath
 import glob
 import duckdb
+from pathlib import Path, PurePath
 
 @pytest.fixture(scope="function")
 def duckdb_empty_cursor(request):
     connection = duckdb.connect('')
     cursor = connection.cursor()
     return cursor
  
+def get_duckdb_root_dir(path_to_search_from):
+    path = Path(path_to_search_from)
+    parents = path.parents
+    for parent in parents:
+        parent_path = PurePath(parent)
```

Comment:
This line looks unnessarary?

Reply:
It's not, I could not get `test_substrait_json.py` to recognize the substrait extension, because it's taking my relative path.
This allows me to run that test from anywhere

Reply:
Really? `parent` should already be a `Path` instance, so I'm not sure what this line is doing